npm/ng-packs/packages/tenant-management/src/lib/components/tenants/tenants.component.ts

```diff
@@ -31,46 +36,36 @@ export class TenantsComponent {
 
   defaultConnectionString: string;
 
-  useSharedDatabase: boolean;
+  _useSharedDatabase: boolean;
```

Comment:
is the first letter necessary? `_`

Reply:
It's necessary, I saw it. But if you are move this variable the 44th line, reading will be easier.